src/manifest/extensions/data/ProspectEventsDataExtension.ts

```diff
@@ -0,0 +1,92 @@
+import { UnknownContextKeys } from '../../../context/keys/UnknownContextKeys';
+import { utils } from '../../../utils';
+import { DataExtension } from './DataExtension';
+import { DataExtensionType } from './DataExtensionType';
+import { ProspectEventsHost } from './ProspectEventsHost';
+
+export class ProspectEventsDataExtension extends DataExtension {
+  /**
+   * Type of mailing link data extension
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#type
+   * @type {DataExtensionType}
+   * @memberof ProspectEventsDataExtension
+   */
+  public type: DataExtensionType.PROSPECT_EVENTS = DataExtensionType.PROSPECT_EVENTS;
+
+  /**
+   * Definition of hosting section
+   *
+   * @type {{ url: string }}
+   * @memberof ProspectEventsDataExtension
+   */
+  public host: ProspectEventsHost;
+
+  /**
+   * Validates the data extension configuration
+   *
+   * @param {Application} application
+   * @return {*}  {string[]}
+   * @memberof ProspectEventsDataExtension
+   */
+  validate(): string[] {
```

Comment:
Some of the paths in this method are untested, examples: "Context key is not one of the valid values", "Context section is not an array" and some others. Please cover all at least untrivial paths. Also slightly confused that the tests are in a file with a different name, not sure if this is common.

Reply:
Tests added for context cases